app/controllers/podcast_metrics_controller.rb

```diff
@@ -71,9 +66,9 @@ def uniques
     @uniques_rollups =
       Rollups::DailyUnique
         .where(podcast_id: @podcast.id, day: (@date_start..@date_end))
-        .select("DATE_TRUNC('#{@interval}', day) AS day, MAX(#{@uniques_selection}) AS #{@uniques_selection}")
-        .group("DATE_TRUNC('#{@interval}', day) AS day")
-        .order(Arel.sql("DATE_TRUNC('#{@interval}', day) ASC"))
+        .select("DATE_TRUNC('#{@uniques_interval}', day) AS day, MAX(#{@uniques_selection}) AS #{@uniques_selection}")
```

Comment:
Uniques on it's own `uniques_interval` param so that it doesn't get updated by the global `interval`.